projects/051 - Webb/template-sql/script.template.sql

```diff
@@ -289,146 +287,103 @@ SELECT
   p.Year,
   p.Month,
   Sex,
-  Ethnic,
+  Ethnicity,
   (p.Year - YearOfBirth) AS Age,
```

Comment:
We now have access to actual date of birth, as RQ062 needed WeekOfBirth. This could be used to calculate  a more accurate age, though I am not sure if we are technically supposed to be using it outside of that study, and I doubt it would make too much difference

Reply:
Yes i don't think it would make much difference, but we should only use that when there is explicit permission.